src/test/java/harness/utils/WorkspaceUtils.java

```diff
@@ -0,0 +1,48 @@
+package harness.utils;
+
+import bio.terra.cli.serialization.userfacing.UFWorkspace;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import harness.CRLJanitor;
+import harness.TestCommand;
+import harness.TestUser;
+
+/** Utilities for working with workspaces in CLI tests. */
+public class WorkspaceUtils {
+
+  /**
+   * Create a new workspace and register it with Janitor if this test is running in an environment
+   * where Janitor is enabled.
+   *
+   * @param workspaceCreator The user who owns the workspace. This user will be impersonated to in
+   *     the WSM workspaceDelete request.
+   */
+  public static UFWorkspace createWorkspace(TestUser workspaceCreator)
+      throws JsonProcessingException {
+    // `terra workspace create --format=json`
+    UFWorkspace workspace =
+        TestCommand.runAndParseCommandExpectSuccess(UFWorkspace.class, "workspace", "create");
+    CRLJanitor.registerWorkspaceForCleanup(workspace, workspaceCreator);
+    return workspace;
+  }
+
+  /**
+   * Create a new workspace and register it with Janitor if this test is running in an environment
+   * where Janitor is enabled.
+   *
+   * @param workspaceCreator The user who owns the workspace. This user will be impersonated to in
+   *     the WSM workspaceDelete request.
+   */
+  public static UFWorkspace createWorkspace(
+      TestUser workspaceCreator, String name, String description) throws JsonProcessingException {
+    // `terra workspace create --format=json --name=$name --description=$description`
+    UFWorkspace workspace =
+        TestCommand.runAndParseCommandExpectSuccess(
+            UFWorkspace.class,
+            "workspace",
+            "create",
+            "--name=" + name,
+            "--description=" + description);
+    CRLJanitor.registerWorkspaceForCleanup(workspace, workspaceCreator);
```

Comment:
(Given that `runAndParseCommandExpectSuccess` and this method return `UFWorkspace`, I think it's ok to use here)

Reply:
Most tests run commands the same way users do (as full commands) rather than calling internal methods, so I think UF object is right here.

(Whether we really should be doing that in unit tests is a whole separate discussion, too...)